pkg/validation/spec/swagger.go

```diff
@@ -46,6 +46,11 @@ func (s Swagger) MarshalJSON() ([]byte, error) {
 	return swag.ConcatJSON(b1, b2), nil
 }
 
+// Uses this library's preferred JSON unmarshaler to decode an instance from JSON
+func (s *Swagger) FromJSON(data []byte) error {
```

Comment:
is `FromJSON` implementing an interface or did we just make up this name?

Reply:
Random name, this is meant to hide the now internal package. Calling decode through the normal `json.Unmarshal` adds an additional layer of computation that doubles the deserizaliation time.

Reply:
but... why not just call `Swagger#UnmarshalJSON(data)`?

Reply:
It looks too simple, we should make it more complicated. /s

Reply:
removed ;)